packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -297,7 +308,10 @@ class MenuContent extends React.Component {
           onKeyDown,
           isActive,
         });
-      // If the child has children then it is an item group, so iterate through it's children
+        if (this.props.headerTitle.length > 0 && item.props.subMenuItems && item.props.subMenuItems.length > 0) {
```

Comment:
Could you add a comment explaining why this check is being done.

Reply:
updated here 16ed11f47288f7ab7e65f9410e955857d99c80f1